virt/release_notes/virt-release-notes-placeholder.adoc

```diff
@@ -1,12 +1,11 @@
 :_content-type: ASSEMBLY
-[id="virt-2-3-release-notes"]
-= {VirtProductName} release notes
+[id="virt-release-notes-placeholder"]
+= {VirtProductName} release notes placeholder
 include::_attributes/common-attributes.adoc[]
-:context: virt-release-notes
-
+:context: virt-release-notes-placeholder
 toc::[]
 
 Do not add or edit release notes here. Edit release notes directly in the branch
 that they are relevant for.
```

Comment:
As mentioned elsewhere, this change applies to main only. Please open a separate PR for it. Thanks!